src/index.html

```diff
@@ -11,7 +11,131 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="test-form"
+    >
+      <fieldset name="presonalInfo" class="form-fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            minlength="2"
+            maxlength="10"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="birth">
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="agreement">
+        </label>
+      </fieldset>
+
+
+      <fieldset name="registrationInfo" class="form-fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            autocomplete="off"
+            required
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset name="interestingInfo" class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input type="radio" name="cats">
```

Comment:
But what about this buttons value? How will it be displayed in forms data that is sent to server?

Reply:
It's hard to say, I have no set up server to check the result of this request. Probably there will be empty value, for 'cats' variable.
`"cats": ""`
But I can definitely say that I should insert value attribute with different values for different radios.

Can I not respond to your remarks if I understand them? Or it is required?:)

Reply:
You may not responding if you agree with comment and understand reason :)
Sometimes questions can specify issue in more clear way, so we using it